src/js/plate-layout.js

```diff
@@ -1,287 +1,331 @@
 $.widget("DNA.plateLayOut", {
```

Comment:
Generally we have been using 2-space indentation in the source files, but this has been switched to 4-space indentation. This makes the diff hard to read. Please reformat this file as 2-space.

Reply:
We have used a tool to reformat all JS files, so that they are indented with 2 spaces. Incidentally, others fixes to formatting were included, so a lot of lines have been changed, but we thought it is for the best. Let us know if you think that is too much and you prefer us to revert.